src/Schema/AST/ASTHelper.php

```diff
@@ -248,13 +248,15 @@ function (DocumentAST $document, ObjectTypeDefinitionNode $objectType) use ($dir
                         return $document;
                     }
 
-                    $objectType->fields = new NodeList(collect($objectType->fields)
-                        ->map(function (FieldDefinitionNode $field) use ($directive): FieldDefinitionNode {
-                            $field->directives = $field->directives->merge([$directive]);
-
-                            return $field;
-                        })
-                        ->all());
+                    $objectType->fields = new NodeList(
+                        collect($objectType->fields)
```

Comment:
Do not use `collect` helper.

Reply:
We can do that in a seperate PR throughout the codebase.

Reply:
https://github.com/nuwave/lighthouse/issues/652

Reply:
Good idea @spawnia